docs/admin/install/vm-install/suse.md

```diff
@@ -26,6 +26,12 @@ For example:
 firewall-cmd --permanent --zone=public --add-service=https
 firewall-cmd --reload
 ```
+-Download the GPG key zip file , unzip and import GPG key
+```
+wget https://github.com/GluuFederation/flex/files/11814579/automation-jans-public-gpg.zip
+unzip automation-jans-public-gpg.zip
+rpm -import automation-jans-public-gpg.asc
```

Comment:
Consider adding a step that actually says that `signature is good`. And ask user to confirm using this message.

If signature verification happens during install itself then installation command should fail in case when signature doesn't match.

Reply:
basically, something like this needs to be added.

```
$ gpg --verify myfile.java.asc 
gpg: assuming signed data in 'myfile.java'
gpg: Signature made mié 23 jun 2021 16:06:54 -05
gpg:                using RSA key CA925CD6C9E8D064FF05B4728190C4130ABA0F98
gpg: Good signature from "Central Repo Test <central@example.com>" [ultimate]
```

Reply:
while installing flex we are getting signature verfication failed error
so we have to manually import gpg key to resolve issue, after importing gpg key issue is resolved.

Reply:
Got it. So Zypper is verifying the signature. :+1: